Basic Stock Fetching

SteelPact opened the pull request:
Simple command for fetching stocks from yfinance and using pandas

CuboidRaptor committed:
f*ck moderation

it's built into discord why the hell would I have it

CuboidRaptor committed:
I dumb

SteelPact committed:
Enoch Commits

Stock Price Stuff

CuboidRaptor commented:
you spelled `yfinanace` wrong in the requirements.txt, btw if the slugsize is too big then Heroku will yell at me so is it possible to reduce the size of packages and stuff like that?
iirc pandas is sorta big

CuboidRaptor committed:
more bugfixxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-

-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

CuboidRaptor committed:
can I just not put a commit message?

well yes but no

CuboidRaptor committed:
some updatessssssssss

um yes (please remind me why my commit messages are such nonsense)

CuboidRaptor committed:
gawd damit forgot to save

CuboidRaptor committed:
thought requests was default

this is a demonstration of my idiocy

CuboidRaptor committed:
i m bad

CuboidRaptor committed:
heroku just work it's not hard

EnochEnzeLi commented:
to bad hmmm actually wait

EnochEnzeLi commented:
no way I don't know how to use numpy other possiblities. So yeah

EnochEnzeLi commented:
And I am guess you to lazy to do anything

EnochEnzeLi commented:
MERGE NOW

EnochEnzeLi commented:
pwease
